.gitignore

```diff
@@ -1 +1,4 @@
-build*/
+build/
+faiss*
```

Comment:
I don't follow the need for these `.gitignore` changes. With this, a `git status` will not show the dirty state of the repository in these directories.

I don't think that is right.